Development

asiimwedismas opened the pull request:
Please review this code so we can merge it to the master branch

We couldn't;t figure out how to host a live demo, maybe that's something u will shade some light one

asiimwedismas committed:
Added the basica html structure layout out

asiimwedismas committed:
i had forgotten to save changes

chirchir12 committed:
added youtube logo, menu, search input, and sign on the header

chirchir12 committed:
added yotube logo, menu, search input and sign in on the header

asiimwedismas committed:
Added the video, video-stats-summary more-more actions, video-description, show-more and comments sections

chirchir12 committed:
added video list on the sidebar

asiimwedismas committed:
added footer content

chirchir12 committed:
refactor and validate html elements

asiimwedismas committed:
Added index css file

asiimwedismas committed:
Refactored the index.html

Problem
the aside wasn't floating correctly to the right in its intial position

Solution
rearrange and moved aside above the main section

asiimwedismas committed:
Positioned the different elements on the page

chirchir12 committed:
refactored video content thumbnails from list item to anchor tag

chirchir12 committed:
added styling on aside-video-list

asiimwedismas committed:
stash changes

asiimwedismas committed:
Merge branch 'development' of https://github.com/asiimwedismas/youtube-clone into development

asiimwedismas committed:
Added header, footer and more actions styles

chirchir12 committed:
styled video-detail section on index.html

asiimwedismas committed:
Added linters

asiimwedismas committed:
updated the readme with project details

addod19 commented:
> Please review this code so we can merge it to the master branch
> 
> We couldn't;t figure out how to host a live demo, maybe that's something u will shade some light one

Navigate to the index.html page on your Github repo and copy the link to this link http://raw.githack.com/. The link will generate development and production link. You can use the development link for your README file live link

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

addod19 commented on index.html:
```diff
@@ -0,0 +1,188 @@
+
+    <main>
+
+        <aside id="thumbnails-aside">
```
The ```<aside>``` is not to be nested in the main. The aside should take the video previews and the main should have the main video together with the other contents

addod19 reviewed (changes requested):
## Status: Require changes ♻️ 🚥 

Hi **TEAM**, you have done a great job with this project 👍 Let us improve our project before we move ahead. Kindly address the comments below: 😄 
* The nav is missing 3 images at the rightmost part of the nav
![im](https://user-images.githubusercontent.com/46388113/77586222-b3b47f80-6edd-11ea-8e00-3be51a47eec0.PNG)
* Kindly edit the PR so it describes the purpose of the project or what exactly you did
* We have 1 warning returned by w3validator
* The main video is not playing. It should play
* The overall UI of the project needs improvement. Check this [link](youtube.com) as a reference

Make these changes and submit another code review 🙂 
_Hapy coding_ 👨‍💻 
Daniel Larbi Addo

asiimwedismas committed:
Added a link working link to the video src attri

Problem
The video in the previous link was not playing, review suggested it
should play.
Also Chnaged the height of the video element to a min-height

asiimwedismas committed:
Added 2more icons in the header to the right

asiimwedismas committed:
Fixed raised issues byt the reviewer

Added 2 more icons in the nav
touched up a lilltle bit on the overall interface
Added a link to a video that works
Fixed the warning from the validator

asiimwedismas commented:
Worked on the issues raised @addod19

asiimwedismas committed:
Upated live demo link

asiimwedismas commented:
The video doesn't seem to be loading on githack but it does on our development machines 
I have also noticed the youtube logo in footer isn't loading but once again, it loaded fine on our development machines

MarioDena reviewed (approved):
Congratulations on implementing the requested changes!

## Project approved! 🥇 

## Code Review by Mario Enrique Delgado Naranjo
